src/Card.js

```diff
@@ -5,36 +5,42 @@ import star from './images/star.png';
 import close from './images/close.png';
 
 class Card extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  deleteCard = (event) => {
+    event.preventDefault();
+    this.props.deleteCard(this);
+  }
+
   render() {
     return (
       <div className ="card">
         <header className="card-header">
           <input className="star-button" type="image" src={star} />
-          { /*<img className="country-flag" src=
-          {canada}/> */ }
           <div className="country-info">
             <h1 className = "country-name">
-            Canada
+            {this.props.country.name}
```

Comment:
You could destructure all of these prop values at the very top of your render method by doing something like:

`let { country, continent } = this.props`

This would allow you to simplify your code when you try to go and render that information in your JSX to:

`{country.name}`
`{continent.name}`
`{country.population}`

etc.